worker/src/handlers/post.ts

```diff
@@ -73,7 +92,22 @@ const sanitizePayload = (
     throw new Error("Missing required keys in the payload");
   }
 
-  if (!InstallationTypes.includes(payload.installation_type)) {
+  if (String(payload.uuid).length !== 32) {
```

Comment:
You still need to make sure that `payload.uuid` is a string

Reply:
```suggestion
  if (typeof payload !== "string" || payload.uuid.length !== 32) {
```

Reply:
We don't use it after this point

Reply:
Are you saying that at this point the unvalidated version was already used to fetch the metadata/value?

Reply:
Yes but no, the storage key is already set, but I can moe it around, it will not be executed anyway when this fails